### ISSUE TITLE
Datagrid column toggle dropdown cut off by app header

### ISSUE THREAD

**Issue by yaooluu** [vmware/clarity#3104]
## Describe the bug

Datagrid column toggle dropdown is blocked by app header.

<img width="781" alt="screen shot 2019-02-13 at 5 33 00 pm" src="https://user-images.githubusercontent.com/8813134/52706628-daa54000-2fc0-11e9-9518-47119dfbbfae.png">

## How to reproduce

https://stackblitz.com/edit/clarity-dark-theme-v1-0-esa7dd

1.  Click the column toggle icon.
2.  Observe that the dropdown is visually blocked by app header.

## Expected behavior

The dropdown should either change direction or float on top of header.

## Versions

**App**

* Angular: 6.0.9
* Clarity: 0.13.x and 1.0

**Device:**

* Type: MacBook
* OS: Mac OS Mojave
* Browser: Chrome
* Version: Latest

**Comment by youdz** [comment]
Duplicate of #2683, please follow that issue for updates.
